src/index.css

```diff
@@ -60,3 +62,50 @@ body {
 .btn-solid-secondary:hover {
   background-color: var(--COLOR-SECONDARY-V3);
 }
+
+.icon-btn-wrapper {
+  position: relative;
+}
+
+.icon-btn-wrapper .name {
+  display: inline-block;
+  opacity: 0;
+  background-color: var(--COLOR-SECONDARY-V3);
+  position: absolute;
+  top: 3rem;
+  right: 0;
+  min-width: 100%;
+  padding: 0 4px;
+  text-align: center;
+  transition: opacity 0.6s ease;
+}
+
+.icon-btn-wrapper:hover .name {
+  opacity: 1;
```

Comment:
What are you trying to do here default opacity is 1

Reply:
initially, I have kept opacity as 0 and on hovering I am making it 1